firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/ApkUpdater.java

```diff
@@ -306,10 +297,7 @@ private void setUpdateTaskCompletionErrorWithDefault(
 
   @VisibleForTesting
   void postUpdateProgress(
```

Comment:
Is this method no longer being tested in this refactor? if so why was that the case and should this then be package-private without the visibleForTesting annotation?

Reply:
Ah, yeah good catch. Yeah before this method was only made visible for testing so that the actual notification display functionality could be tested. That functionality is now covered by AppIconSourceTest and FirebaseAppDistributionNotificationsManagerTest.

I've made it private and removed the annotation.